Unbound Reality/Assets/Hub World/Scripts/Pig/Pig_Controls.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Pig_Controls : MonoBehaviour {
+    public float moveSpeed = 10f;
+    public float turnSpeed = 50f;
+
+    private float fob, rol;
+    // Use this for initialization
+    void Start () {
```

Comment:
I'll merge as is, but In the future, if a function exist without doing anything just delete it.